spec/models/match_spec.rb

```diff
@@ -45,5 +45,7 @@
 
   it 'has a valid factory' do
     expect(build(:match)).to be_valid
+    expect(build(:running_playoff_match)).to be_valid
```

Comment:
use [factory linting](https://www.rubydoc.info/gems/factory_bot/file/GETTING_STARTED.md#Linting_Factories) instead.

Reply:
I do not know how to do that, i tried to but failed :D (The documentation says It should not be done before the test suite, but rather as a task. So i tried to implement a rake task which doesn't seem to work; checkout this [commit](https://github.com/turniere/turniere-backend/commit/a0677a2644e43273a2023cb4bc91059627dd3c69)

Reply:
What doesn't work? The rake task you added runs without errors on my system.

Reply:
Yes it does, but it will run flawless even if the Factory is invalid..

Reply:
fixed